terraform/modules/frontdoor/variables.tf

```diff
@@ -1,3 +1,7 @@
+variable "ssl_certs_created" {
+  description = "Indicates whether ssl certificates have already been manually created"
```

Comment:
do we have docs somewhere on what this creation process should look like?

Reply:
Good point. There is a TODO in the ReadMe for this - I'll update that as part of this set of changes.